generate-docker-image-tags/action.yml

```diff
@@ -60,9 +60,21 @@ runs:
           TAGS="${{ inputs.prefix }}.$BRANCH,${{ inputs.prefix }}.$COMMIT_SHA"
         fi
 
+        if [ "$GITHUB_EVENT_NAME" == "pull_request" ]
+        then
+          PR_NUMBER=${{ github.event.number }}
+        else
+          # Workflow was triggered by a "push" event.
+          PR_NUMBER=$(
+            echo "${{ github.event.head_commit.message }}" \
```

Comment:
This looks good, but does it require that the PR merge commit needs to contain the sc story id in order for this to work?